File: Faltantes o por revisar/Generadores de casos de prueba/TestCasesGeneratingRandomMatrix.cpp
#include <cstdlib>
#include <bits/stdc++.h> 
using namespace std;

int RUN = 5;
int lowerBound = 0;
int upperBound = 1000;
int maxColumn = 10;
int minColumn = 1;
int minRow = 1;
int maxRow = 10;

void randomMatrix() {
    for (int i = 0; i < RUN; i++) {
        int row = maxRow - (rand() % maxRow) - minRow;
        int column = maxColumn - (rand() % maxColumn) - minColumn;
        int matrix[row][column];
        cout << row << " " << column << endl;
        for (int j = 0; j < row; j++) {
            for (int k = 0; k < column; k++) {
                int a = upperBound - (rand() % upperBound);
                cout << a << " ";
            }
            cout << endl;
        }
        cout << endl;
    }
}

int main() {
    randomMatrix();
}

File: Recien traducido/Generadores de casos de prueba/TestCasesGeneratingRandomMatrix.py
import random as r
RUN = 5;
lowerBound = 0
upperBound = 1000
maxColumn = 10
minColumn = 1
minRow = 1
maxRow = 10
def randomMatrix():
    for i in range (RUN):
        row = r.randint(minRow,maxRow)
        column = r.randint(minColumn,maxColumn)
        matrix=[[0 for x in range(row)]for y in range (column)];
        print(str(row) + " " + str(column))
        str1=""
        for i in range (row):
            for j in range (column):
                a = r.randint(0,300)
                str1+=str(a)+" " 
            print(str1)
            str1=""
randomMatrix()
